google/cloud/bigquery/job/base.py

```diff
@@ -202,6 +202,16 @@ def script_statistics(self) -> Optional["ScriptStatistics"]:
             return None
         return ScriptStatistics(resource)
 
+    @property
+    def session_info(self) -> Optional["SessionInfo"]:
+        """[Preview] Information of the session if this job is part of one."""
```

Comment:
Do we add the versionadded notice to the docstring for preview properties?
It would probably also be nice to include a URL to the product reference like we already do in several other classes.

(applies to the other new properties as well)